dist/schema.json

```diff
@@ -144,6 +144,9 @@
           "url": "https://docs.joinmastodon.org/methods/accounts/#create",
           "description": "Official Mastodon API documentation"
         },
+        "x-badge": {
```

Comment:
Don't mark existing operations with changes as unreleased.

For example `POST /api/v1/accounts` lists `4.4.0 - added date_of_birth parameter` so the method got a new parameter but is not new itself.